### PR TITLE
Fix cycle waypoint position

### PR DESCRIPTION
**Pull request by oOKexOo**
**When merged this pull request will:**
- Fix https://feedback.bistudio.com/T85817

### CODE REVIEW

File: addons/editor/functions/fnc_bi_curatorWaypointPlaced.sqf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
- * Author: Bohemia Interactive, mharis001
+ * Author: Bohemia Interactive, mharis001, Kex
  * Handles placement of a waypoint by Zeus.
- * Edited to allow control over radio messages.
+ * Edited to allow control over radio messages and fixing waypoint positioning.
```
Comment (CreepPork) [inline addons/editor/functions/fnc_bi_curatorWaypointPlaced.sqf]:
```suggestion
 * Edited to allow control over radio messages and fixed cycle waypoint position.
```

File: addons/editor/functions/fnc_bi_curatorWaypointPlaced.sqf [hunk 2]
```diff
@@ -18,11 +18,23 @@
...
+
+if (GVAR(unitRadioMessages) != 2) then {
+    [
+        _leader,
+        ["CuratorWaypointPlaced", "CuratorWaypointPlacedAttack"] select (_waypointType == "DESTROY")
+    ] call BIS_fnc_curatorSayMessage;
+};
+
+if (_waypointID == 1) then {
+    [_group, 0] setWaypointPosition [getPos _leader, 0];
```
Comment (mharis001) [inline addons/editor/functions/fnc_bi_curatorWaypointPlaced.sqf]: `getPos` returns AGLS, `setWaypointPosition` expects AGL or Object. Can pass the leader directly here.

Do we want to force exact position here? If so, then this should use ASL with a -1 radius.

File: addons/editor/functions/fnc_bi_curatorWaypointPlaced.sqf [hunk 3]
```diff
@@ -18,11 +18,23 @@
...
+};
+
+if (_waypointType == "CYCLE") then {
+    _waypoint setWaypointPosition [waypointPosition [_group, 0], 0];
```
Comment (mharis001) [inline addons/editor/functions/fnc_bi_curatorWaypointPlaced.sqf]: Should this also force exact position with -1 as the radius?

File: addons/editor/functions/fnc_bi_curatorWaypointPlaced.sqf (hunk 2 above)
Comment (oOKexOo) [reply addons/editor/functions/fnc_bi_curatorWaypointPlaced.sqf]: Yes, let's go for the -1 option.

**Pull request merged**